code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -23,7 +23,7 @@ Borg Hypospray
 	var/recharge_time = 5 //Time it takes for shots to recharge (in seconds)
 
 	var/list/datum/reagents/reagent_list = list()
-	var/list/reagent_ids = list("salbutamol", "salglu_solution", "charcoal", "epinephrine", "spaceacillin")
+	var/list/reagent_ids = list("reoxygenator", "aloe_vera_extract", "toxin_filterer", "brute_solution", "epinephrine", "spaceacillin", "stabilized_omnizine")

```

Comment:
Adding "stabilised" after any of the dangerous chemicals you don't feel like having be dangerous right now is lazy